cypress/integration/tag_amd_spec.js

```diff
@@ -0,0 +1,610 @@
+            cy.get('.tags .tag').should('have.length', 2);
+        });
+
+        it('should do nothing when backspace is typed and there are no tags', () => {
+            cy.get('.tags .tag').should('have.length', 0);
+            cy.get('#tags').type(' {backspace}');
+            cy.get('.tags .tag').should('have.length', 0);
+            cy.get('.tags .tag').should('not.have.class', 'selected');
+        });
+
+        it('should do nothing when left arrow is typed and there are no tags', () => {
+            cy.get('.tags .tag').should('have.length', 0);
+            cy.get('#tags').type(' {leftarrow}');
+            cy.get('.tags .tag').should('have.length', 0);
+            cy.get('.tags .tag').should('not.have.class', 'selected');
+        });
+
+        it('should add all tags in the input when comma is typed', () => {
+            cy.get('.tags .tag').should('have.length', 0);
+            cy.get('#tags').type('tag1, another tag');
+            cy.get('.tags .tag').should('have.length', 1);
+            cy.get('#tags').type(',');
+            cy.get('.tags .tag').should('have.length', 2);
+        });
+
+        it('should add all unique tags in the input when comma is typed and there are repeated tags', () => {
+            cy.get('.tags .tag').should('have.length', 0);
+            cy.get('#tags').type('tag1, another tag, tag1, yet another');
+            cy.get('.tags .tag').should('have.length', 2);
+            cy.get('#tags').type(',');
+            cy.get('.tags .tag').should('have.length', 3);
+        });
+
+        it('should add tag when key and comma is typed', () => {
+            cy.get('.tags .tag').should('have.length', 0);
+            cy.get('#tags').type('tag1');
+            cy.get('.tags .tag').should('have.length', 0);
+            cy.get('#tags').type(',');
+            cy.get('.tags .tag').should('have.length', 1);
+            cy.get('#tags').type('A');
+            cy.get('.tags .tag').should('have.length', 1);
+            cy.get('#tags').type(',');
+            cy.get('.tags .tag').should('have.length', 2);
+        });
+    });
+
+    context('Selecting tags of a tagified input', () => {
+        beforeEach(() => {
+            cy.visit('/?test=selecting');
+        });
+
+        it('should have no selected tags after activated', () => {
+            cy.get('#tags').type('tag1, tag-to-remove{enter}');
+            cy.get('.tags .tag').should('have.length', 2);
+            cy.get('.tags .tag.selected').should('have.length', 0);
+        });
+
+        it('should not select tags when pressing left key if input is filled', () => {
+            cy.get('#tags').type('tag1, another{enter}');
+            cy.get('#tags').type('tag2{leftarrow}');
+            cy.get('.tags .tag.selected').should('have.length', 0);
+        });
+
+        it('should not select tags when pressing right key if input is filled', () => {
+            cy.get('#tags').type('tag1, tag-to-remove{enter}');
```

Comment:
The input is not filled after this point, so the test is not doing what it should.

Reply:
Hum, I don't know. The difference in this test it I don't had use the method directly... this is a problem?